datajunction-server/datajunction_server/models/node.py

```diff
@@ -473,8 +476,11 @@ class MetricMetadataInput(BaseModel):
     Metric metadata output
     """
 
-    direction: Optional[MetricDirection]
-    unit: Optional[str]
+    direction: MetricDirection | None
+    unit: str | None
```

Comment:
`str` or `Unit` ?

Reply:
In this case it has to be `str` because pydantic won't interpret a complex enum like `Unit` correctly